docs/howto/add_relationship_links.md

```diff
@@ -15,40 +15,42 @@ class Api::V1::UsersController < ApplicationController
       serializer: Api::V1::UserSerializer,
       include: []
   end
+end
 ```
 
 Bear in mind though that ActiveModelSerializers are [framework-agnostic](outside_controller_use.md), Rails is just a common example here.
 
 ### Links as an attribute of a resource
-**This is applicable to JSONAPI, JSON and Attributes adapters**
+**This is applicable to JSON and Attributes adapters**
 
 You can define an attribute in the resource, named `links`.
 
 ```ruby
 class Api::V1::UserSerializer < ActiveModel::Serializer
-  attributes :id, :name, :links
+  include Rails.application.routes.url_helpers
```

Comment:
It is *working* for all the adapter but not *correct* for JSONAPI as the spec forbid to have an attribute named `links`:

> The value of the attributes key MUST be an object (an “attributes object”). Members of the attributes object (“attributes”) represent information about the resource object in which it’s defined.
>
> Attributes may contain any valid JSON value.
>
> Complex data structures involving JSON objects and arrays are allowed as attribute values. However, any object that constitutes or is contained in an attribute MUST NOT contain a relationships or links member, as those members are reserved by this specification for future use.

Also I had the url helper because within the `links` method there was a call to `api_v1_user_path`. And that call would not be correct without the url helper. I simply "fixed" the existing doc as I did not want to modify it as to avoid having too many changes.